valve-monitor/web-app/src/pages/api/clear-alarms.ts

```diff
@@ -0,0 +1,29 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import { ErrorWithCause } from "pony-cause";
+import { HTTP_STATUS } from "../../constants/http";
+import { services } from "../../services/ServiceLocatorServer";
+
+export default async function clearAlarmsHandler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  switch (req.method) {
+    case "POST":
+      try {
+        await services().getAppService().clearAlarms();
+      } catch (cause) {
+        throw new ErrorWithCause("Could not clear alarms", {
+          cause,
+        });
+      }
+
+      // res.status(StatusCodes.OK).json({});
```

Comment:
remove commented out code

Reply:
Ah, I accidentally committed my code to test the error scenario 😬 Good catch.